static/js/scratch.js

```diff
@@ -16,172 +16,146 @@ var yuiOptions = {
+              lastNode.addClass('last-item');
+            }
+          }
+          lastNode = node;
+        });
+        if(lastNode){
+          lastNode.addClass('last-item');
+          y = 0;
+          lastNode = null;
+        }
+      });
+    }, 1000);
+  };
 
-core.homeAnimation = function() {
-  if(Y.one('body').hasClass('home')){
-    var anim = Y.one('.animation');
-    if(anim !== null) {
-      anim.addClass('run');
+  loadRSSFeed = function(id, tag, limit, url, title) {
+    var feedURL = "https://insights.ubuntu.com/feed/";
+    if (tag) {
+      feedURL += "?tag=" + tag;
     }
-  }
-};
-
-core.svgFallback = function() {
-  if (!Modernizr.svg || !Modernizr.backgroundsize) {
-    Y.all("img[src$='.svg']").each(function(node) {
-      node.setAttribute("src", node.getAttribute('src').toString().match(/.*\/(.+?)\./)[0]+'png');
-    });
-  }
-};
 
-core.autoLastItem = function() {
-  var y = 0,
-    lastNode = null;
-  setTimeout(function(){
-    Y.all('.list-auto').each(function(node) {
-      node.all('li').each(function(node) {
-        y = node.getXY()[1];
-        if(lastNode) {
-          ly = lastNode.getXY()[1];
-          if(y > ly + 50) { //Added +50 to stop differing-height images breaking this
-            lastNode.addClass('last-item');
-          }
+    if (limit) {
+      var feedLimit = limit | 5;
+    }
+    $.getFeed({
+      url: feedURL,
+      success: function(feed) {
+        var html = "<ul class='list'>";
+        for(var i = 0; i < feed.items.length && i < feedLimit; i++) {
+          var item = feed.items[i];
+          html += "<li><a href='" + item.link + "'>" + item.title + "&nbsp;&rsaquo;</a></li>";
         }
-        lastNode = node;
-      });
-      if(lastNode){
-        lastNode.addClass('last-item');
-        y = 0;
-        lastNode = null;
+        html += "</ul>";
+        if ($('#' + id)) {
+          $('#' + id).append(html);
+        }
+      }, failure: function () {
+
       }
     });
-  }, 1000);
-};
-
-loadRSSFeed = function(id, tag, limit, url, title) {
-  var feedURL = "https://insights.ubuntu.com/feed/";
-  if (tag) {
-    feedURL += "?tag=" + tag;
   }
 
-  if (limit) {
-    var feedLimit = limit;
-  } else {
-    var feedLimit = 5;
-  }
-  $.getFeed({
-    url: feedURL,
-    success: function(feed) {
-      var html = "<ul class='list'>";
-      for(var i = 0; i < feed.items.length && i < feedLimit; i++) {
-        var item = feed.items[i];
-        html += "<li><a href='" + item.link + "'>" + item.title + "&nbsp;&rsaquo;</a></li>";
-      }
-      html += "</ul>";
-      if ($('#' + id)) {
-        $('#' + id).append(html);
-      }
-    }, failure: function () {
-
-    }
-  });
-}
-
-core.setupAccordion();
-core.cookiePolicy();
-core.setupHtmlClass();
-core.parallaxBackground();
-core.homeAnimation();
-core.svgFallback();
-core.autoLastItem();
+  core.setupAccordion();
+  core.cookiePolicy();
+  core.setupHtmlClass();
+  core.parallaxBackground();
+  core.autoLastItem();
```

Comment:
Let's lose all this js. Looks like the feeds are via the template.

Reply:
RSS feeds are still loaded with JS on the partner pages